fix issue#162

philcaz opened the pull request:
Fix HTTP Errorr 403 by defining user agent that mimics browser request.

Fix relative path to journal_abbreviations_mathematics.csv

Phikho-cc committed:
fix issue#162

Fix HTTP Errorr 403 by defining user agent that mimics browser request.

Fix relative path to journal_abbreviations_mathematics.csv

surluna commented:
Hi Phil, thank u for fixing the issue. 
Setting `headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36'}` to mimic seems like a practical solution for 403 error. But as the website changes, it might need updating, so it’s a temporary but effective solution.
And `print(os.getcwd())` is useful for debugging but should be removed from the final version unless there’s a need to frequently monitor the directory.
The path update looks good and solves potential issues with relative paths, great job!

philcaz commented:
Thanks for the review, I've made the changes accordingly. Please let me know if further refinement is needed

surluna commented:
Great, we are good to go now.

Phikho-cc committed:
remove debug print and consolidate header

Remove the print(os.getcwd()) statement for path debugging.
Consolidate user request header by mimicing google chrome
